crates/sui-core/src/authority_active/gossip/mod.rs

```diff
@@ -297,6 +281,14 @@ where
                                 self.state.metrics.gossip_queued_count.inc();
 
                             }
+                            match self.max_seq {
```

Comment:
Are you sure this is correct: if we observe an item, and then drop the connection; then reconnect we might legitimately be sent the same item, since we already start at the (potentially previous) batch. So at best i think we cen track the sequence number of batches, but not items -- because these may be repeated for the last batch across two separate requests.